### PR TITLE
Feature list structure for To do list

### PR DESCRIPTION
**Pull request by gitbygaurav**
## **On this feature branch, the below features have been implemented :-

- Created an HTML file and added their markup.
- Created an empty to do list placeholder in the HTML file.
- Seted the HTML file as a template using the HTML Webpack Plugin.
- Created a Js file and seted an array of objects of some simple to do tasks.
- Each task object contains three keys- 
  - Description [string]
  - Completed [boolean]
  - Index [number]
 - Written a function that iterate over the tasks array and populate an HTML list item element for each task.
 - Created a CSS file and set the rules for the To Do List.

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+main {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.list {
+  width: 60%;
+  margin-top: 100px;
+}
```
Comment (AmaduKamara) [inline src/style.css:19]: ## [ Optional Change ] 
You did well on your styles but you could add a little more style to make things look elegant and match the original design by giving the todo an elevated box shadow.

- [ ] I highly recommend you work on this change to improve the look of your app by adding this style to your `.list` class `box-shadow: 1px 4px 30px rgba(0, 0, 0, 0.1);`. You're welcome 👍 



**See the original design**
image

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢 

Hi @gauravgangwarweb,

Your project is complete!  

## Highlights:
✅ You used Gitflow correctly 👍 
✅ Linter checks passed 👍 
✅ Good commit messages 👍 
✅ Good descriptive PR 👍 
✅ Todo app is structured well as expected 👍 

There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/style.css (hunk 1 above)
Comment (gauravgangwarweb) [reply src/style.css:19]: Thanks 😊,  for approval, I will work on optional changes.

**Comment by gitbygaurav** [comment]
> # Status: Approved 🟢🟢🟢
> Hi @gauravgangwarweb,
> 
> Your project is complete!
> 
> ## Highlights:
> ✅ You used Gitflow correctly 👍 ✅ Linter checks passed 👍 ✅ Good commit messages 👍 ✅ Good descriptive PR 👍 ✅ Todo app is structured well as expected 👍
> 
> There is nothing else to say other than... it's time to merge it: ship it: Congratulations! 🎉
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Thanks 😊,  for approval, I will work on optional changes.

**Pull request merged**